eng/pipelines/templates/steps/template-test-run-job.yml

```diff
@@ -4,6 +4,16 @@
 # and additional variables specified by template-test-generate-jobs.yml
 
 steps:
+# TODO: Pre-install tools should only run if the tools are not already 
+# installed (i.e. only on ARM64 images)
+- bash: lsb_release -a && uname -a && eng/scripts/preinstall-tools.sh
+  displayName: Pre-install tools 
+# TODO: This may only need to be run if we're in ARM64 Linux
```

Comment:
FWIW - One of the integration tests for `azd` uses this - but that integration tests is also disabled right now because it was flaky.

You can remove the call to `t.Skip()` here: https://github.com/Azure/azure-dev/blob/b6c8218e832d75a5eb5d0915016390347d3be29f/cli/azd/test/functional/cli_test.go#L221 to have the integration test run on your PR.